### ISSUE TITLE
can't identify element: table is in form of “Wpfobject”

### ISSUE THREAD

**Issue by amalllm** [microsoft/WinAppDriver#1534]
I'm using winappdriver to automate window desktop application (wpf/infragistics).
i try to identify table elements using Inspet.exe, but it didn't work. because Tables are not in form of “Wpftable”, Instead they are in form of “Wpfobject” And “WpfEdit”.
Do you know how to manage this issue please?
Do you think that winappdriver is not compatible with py application? if not  do you have any idea about another  driver that is compatible with WPF/Infragitics 
Thanks for your help

**Comment by anunay1** [comment]
if inspect does not identify the elements then Windows Application Driver will not work.

**Comment by anunay1** [comment]
if the application is under your control, ask the developers to implement automation properties.

**Comment by amalllm** [comment]
thank a lot for your response. But the inspect return for all the elements the same properties ( except mouse move) :
Name:	""
ControlType:	UIA_TabControlTypeId 
LocalizedControlType:	"tab"
AutomationId:	"DynamicTab"
FrameworkId:	"WPF"
ClassName:	"TabGroupPane"
So i can't identify the butons and editbox  ...

**Comment by anunay1** [comment]
Yes because there are no Automation properties implemented for elements.

**Comment by amalllm** [comment]
ok i see :) ,the application is not under of my control, so i want to know please if there is another solution to handle this problem? or may be using another tool?
thank you for your help

**Comment by anunay1** [comment]
Check out paid tools like UFT, Test Complete
